### ISSUE TITLE
[SCREENREADER]: For the current page CONSIDER adding 'aria-current="page"` attribute to the sidenav link

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8964]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

**Consider** providing an indicator of current page status in the navigation. Without an indicator, it could be confusing, lead to repeatedly selecting the page one wants to go to, despite being on it, since other cues are not announced to the screen reader user.

> It is common on the web for the current thing in a collection to be highlighted visually, but providing an alternative for screen reader users has often involved something of a hack. The **aria-current attribute** is intended to solve this problem.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user I want to know when I am on the current page in the navigation.

## Environment

* Operating System: any
* Browser: any
* Screenreading device: any
* Server destination: https://staging.va.gov/pittsburgh-health-care/

## Steps to Recreate

1. Enter https://staging.va.gov/pittsburgh-health-care/ in browser
1. Start screenreading device listed in Environment
1. Navigate to the side navigation
1. Select Health services to navigate to that page
1. Have the screen reader read the page, verify that there is no audible indication of current page
1. Hover over Health services in the side nav, verify that it displays as interactive
1. Select the link, verify that it reloads the page

## Possible Fixes (optional)

Add `aria-current="page"` to the href for the current page so that screen readers will indicate it is the current page. This link should also be not be clickable.

### Current code
```diff
<style>
.va-sidenav .open {
    color: #323a45!important;
    background: linear-gradient(#323a45,#323a45) 0/8px 100%,#f8f8f8;
    background-repeat: no-repeat;
}
</style>
<li class="va-sidenav-level-2 active">
!  <a aria-label="Health services" class="va-sidenav-item-label open" href="/pittsburgh-health-care/health-services" rel="noopener noreferrer" style="padding-left: 20px;">
    <span class=""> 
      Health services 
    </span>
  </a>
</li>
```

### Recommended code
- Moved the padding to a style rather than inline CSS
- Added `aria-current="page"` to the href attributes
- Modified the CSS pointer-events and cursor to make the link not clickable

```diff
<style>
.va-sidenav-level-2 a {
    padding-left: 20px; }
.va-sidenav .open {
    color: #323a45!important;
    background: linear-gradient(#323a45,#323a45) 0/8px 100%,#f8f8f8;
    background-repeat: no-repeat;
+  pointer-events: none;
+  cursor: default;
}
</style>
<li class="va-sidenav-level-2 active">
+  <a aria-current="page" aria-label="Health services" class="va-sidenav-item-label open" href="/pittsburgh-health-care/health-services" rel="noopener noreferrer">
    <span class=""> 
      Health services 
    </span>
  </a>
</li>
```

## WCAG or Vendor Guidance (optional)

* [Using the aria-current attribute](https://tink.uk/using-the-aria-current-attribute/)
* [W3C aria-current attribute](https://www.w3.org/TR/wai-aria-1.1/#aria-current)

## Screenshots or Trace Logs

![Screen Shot 2020-05-11 at 1 54 56 PM](https://user-images.githubusercontent.com/57469/81603692-c5be8480-939c-11ea-86e7-8c45aa4c4c8b.png)

**Comment by 1Copenut** [comment]
Agree 💯 this is a great addition and semantic foothold for assistive tech.

**Comment by mpelzsherman** [comment]
@jenstrickland - we are already putting an `aria-current="page"` attribute on the current link. However, when I select the currently selected link with a screen reader, I don't hear anything different. Can you please update the AC to indicate what we are supposed to be listening for, and if you have any suggestions for what to change in the code, please let us know, thanks.

![VA_Pittsburgh_Health_Care___Pittsburgh_VA_Medical_Center-University_Dr](https://user-images.githubusercontent.com/80267/94468207-14c15680-0192-11eb-874f-8499e440c7ea.png)

**Comment by jenstrickland** [comment]
@mpelzsherman The `aria-current="page"` was not there at the time of this review. Here's a screenshot of what VoiceOver reads out to communicate the current page.


![Screen Shot 2020-09-28 at 6 19 17 PM (2)](https://user-images.githubusercontent.com/57469/94491964-3c76e580-01b7-11eb-8258-34bb5f9f6efd.png)
